src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -1138,6 +1141,26 @@
   <script src="{{ asset('../js/tiny_mce/tiny_mce.js') }}"></script>
   <script src="{{ asset('../js/admin/tinymce.inc.js') }}"></script>
   <script src="{{ asset('../js/admin/tinymce_loader.js') }}"></script>
+
+  <script>
+      $(function() {
+        var editable = '{{ editable }}'
```

Comment:
```js
var editable = '{{ editable }}'
```

So `editable = 'true'` or `editable = 'false'` ?
So … `editable !== true` && `editable !== false` 

Is it dangerous to just do:

```js
var editable = {{ editable }}
```

?

(Also I think this hack is here because there is no other solution ^^)